index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Holi 2023</title>
+  <link rel="stylesheet" href="./css/index.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+</head>
+<body>
+  <header>
+    <div class="topbar">
+      <ul>
+        <li><a href="#"><i class="bi bi-facebook"></i></a></li>
+        <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
```

Comment:
- [Optional] Dear @Kajal19-del, good job adding the top Navbar. But I suggest you follow up on the font type and the sizing. your design is a bit different from the given template. 

### Figma design 
![](https://user-images.githubusercontent.com/63473584/217944052-118c9fec-9e12-44b4-b9df-0cbd190304a0.png)

### Your design.
![Screen Shot 2023-02-09 at 11 47 07 PM](https://user-images.githubusercontent.com/69077061/218033048-e6a97cbb-fded-410e-a2b0-54b05f4813e0.png)